os/compression/Kconfig

```diff
@@ -17,10 +17,11 @@ if COMPRESSED_BINARY
 config COMPRESSION_TYPE
 	int "Compression Algorithm Type"
 	default 1
```

Comment:
What is the default value for this? not 2?

Reply:
It has not been decided yet whether `LZMA` or `MINIZ` will be default.
Will modify accordingly when decided.

Reply:
When I saw the mail, you guys looks decided already. Could you let me know what you check more?